test/fixtures/templates/bad/noecho.yaml

```diff
@@ -1,13 +1,19 @@
-AWSTemplateFormatVersion: '2010-09-09'
+AWSTemplateFormatVersion: "2010-09-09"
 Parameters:
   NoEchoParam:
     Type: String
     Default: NoEchoWasEchoed
     NoEcho: true
 Resources:
-  SNSTopicWithSecretName:
+  SNSTopicWithSecretNameInRef:
     Type: AWS::SNS::Topic
     Properties:
       DisplayName: testname
     Metadata:
       NoEchoParamInMetadata: !Ref NoEchoParam
+  SNSTopicWithSecretNameInSub:
+    Type: AWS::SNS::Topic
+    Properties:
+      DisplayName: testname
+    Metadata:
+      NoEchoParamInMetadata: !Sub "${NoEchoParam}"
```

Comment:
[Looks like it's catching both as expected.](https://travis-ci.com/github/aws-cloudformation/cfn-python-lint/jobs/361865901#L351) This should be bumped as well now:

https://github.com/aws-cloudformation/cfn-python-lint/blob/b8b6cf1cc4420b28c5cb757dc2377af63b4a1acf/test/unit/rules/resources/test_noecho.py#L22